.github/workflows/wheels.yml

```diff
@@ -81,7 +120,7 @@ jobs:
           - {"os": "ubuntu", "build": "cp310-musllinux_ppc64le", "arch": "ppc64le"}
           - {"os": "ubuntu", "build": "cp310-musllinux_s390x", "arch": "s390x"}
           - {"os": "ubuntu", "build": "cp310-musllinux_x86_64", "arch": "x86_64"}
-          - {"os": "ubuntu", "build": "pp37-manylinux_aarch64", "arch": "aarch64"}
+          # - {"os": "ubuntu", "build": "pp37-manylinux_aarch64", "arch": "aarch64"}
```

Comment:
If this is the one segfaulting, that may deserve a little comment and follow up task to document it

Reply:
Good suggestion.  Comment added.  See failure here:
https://github.com/pytoolz/cytoolz/runs/7262094604?check_suite_focus=true#step:5:307

I suppose any segfault is at least a little worrisome.  This one doesn't worry me that much, but I would be delighted if we figured it out.  Anyway, using `cytoolz` in PyPy for an old version of Python on aarch64 architecture is probably very uncommon.  If you're using aarch64, then you're probably using a newer Python, and if you're using PyPy, you may not even be using `cytoolz` (I don't know the performance difference between `toolz` and `cytoolz` on PyPy).  So, hopefully this was a narrow issue in PyPy that got resolved.

NumPy is dropping 3.7 support in less than 6 months, and CPython is dropping security support for 3.7 in less than 12 months.  We try to support old versions of Python for `toolz` and `cytoolz`, because it's cheap for us to do so.  I personally don't feel compelled to investigate this segfault without a user asking for this wheel.

Reply:
Yeah that makes complete sense indeed, all good points, thanks for the extra context :+1: